src/main/resources/db/changelog/changeset/create-products-table.yaml

```diff
@@ -0,0 +1,45 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-products-table
+      author: anovosiolov@griddynamics.com
+      changes:
+        - createTable:
+            tableName: products
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    unique: true
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: title
+                  type: varchar (255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: available
+                  type: int
+                  constraints:
+                    nullable: false
+              - column:
+                  name: price
+                  type: decimal
+                  constraints:
+                    nullable: false
+
+        - createTable:
+            tableName: products_seq
```

Comment:
If the table is related to id generator in Hibernate, to use table is not recommended because is the less performant, use database sequence instead. The table strategy is used only if the RDBMS doesn't support sequence.

Reply:
conversation resolved